cg/cli/delete/illumina_sequencing_run.py

```diff
@@ -0,0 +1,28 @@
+import click
+
+from cg.constants.cli_options import DRY_RUN
+from cg.services.illumina.post_processing.housekeeper_storage import (
+    delete_sequencing_data_from_housekeeper,
+)
+
+
+@click.command("illumina-run")
+@DRY_RUN
+@click.option(
+    "--flow-cell-id",
+    "flow_cell_id",
+    type=str,
+    multiple=False,
+    help="Give the flow cell id of the run to delete from statusdb and housekeeper.",
+)
+@click.pass_context
+def delete_illumina_run(context: click.Context, dry_run: bool, flow_cell_id: str):
+    """Delete an Illumina sequencing run"""
+    if dry_run:
+        click.echo("Dry run: Would delete the Illumina sequencing run")
+        return
```

Comment:
This dry run does not seem to add any value

Reply:
True I removed it